samples/WorkerBindingSamples/Table/TableInputBindingSamples.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net;
+using Azure.Data.Tables;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+
+namespace WorkerBindingSamples.Table
+{
+    public class TableInputBindingSamples
+    {
+        private readonly ILogger<TableInputBindingSamples> _logger;
+
+        public TableInputBindingSamples(ILogger<TableInputBindingSamples> logger)
+        {
+            _logger = logger;
+        }
+
+        [Function(nameof(TableClientFunction))]
+        public async Task<HttpResponseData> TableClientFunction(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequestData req,
+            [TableInput("TableName")] TableClient table)
+        {
+            var tableEntity = table.QueryAsync<TableEntity>();
+            var response = req.CreateResponse(HttpStatusCode.OK);
+
+            List<string> tableList = new();
+            await foreach (TableEntity val in tableEntity)
+            {
+                val.TryGetValue("Text", out var text);
+                _logger.LogInformation("Value of text: " + text);
+
+                await response.WriteStringAsync(text?.ToString() ?? "");
+            }
+
+            return response;
+        }
+
```

Comment:
Consider adding sample functions for some types which bypass deferred binding and get handled on Extension side, if exist.

Reply:
Added